test/test_utils/__init__.py

```diff
@@ -1126,12 +1126,12 @@ def get_inference_run_command(image_uri, model_names, processor="cpu"):
         # PyTorch 2.4 requires token authentication to be disabled.
         _framework, _version = get_framework_and_version_from_tag(image_uri=image_uri)
         auth_arg = (
-            " --disable-token-auth"
+            "--disable-token-auth"
             if _framework == "pytorch" and Version(_version) in SpecifierSet(">=2.4")
             else ""
         )
         mms_command = (
-            f"{server_cmd} --start{auth_arg} --{server_type}-config /home/model-server/config.properties --models "
+            f"{server_cmd} --start {auth_arg} --{server_type}-config /home/model-server/config.properties --models "
```

Comment:
nit: in the case if this is empty string, there are two spaces in this string

Reply:
yeah that's why I initially did the `--start{auth_arg}`. But I checked in docker command locally and double spaces don't affect anything in that order since the flag doesn't accept following string. I only made that change just for tidiness but if you prefer to have one space separation I can change it back.

Reply:
I think this is a bit subjective, let's just leave as is, as we follow similar patterns in other parts of the code